src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -60,7 +61,6 @@
 use Symfony\Component\Security\Core\User\UserProviderInterface;
 use Symfony\Component\Security\Http\Authenticator\Debug\TraceableAuthenticatorManagerListener;
 use Symfony\Component\Security\Http\Event\CheckPassportEvent;
-use Symfony\Flex\Command\InstallRecipesCommand;
 
 /**
  * SecurityExtension.
```

Comment:
Can be removed IMHO

Reply:
@OskarStark are you referring to the "SecurityExtension." comment? That wasn't added by this PR. It seems to have been present since the file was created in https://github.com/symfony/symfony/commit/f216f313e8909015fe9961a5604d179f64c35a90 15 years ago.

It's present in other bundle extension classes too, e.g:

https://github.com/symfony/symfony/blob/79fa5f2911946c739d80d4a89d8cdebbfe80c2dd/src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php#L23
https://github.com/symfony/symfony/blob/79fa5f2911946c739d80d4a89d8cdebbfe80c2dd/src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php#L39

So I think it's beyond the scope of this PR to remove it.